docs/02-app/01-building-your-application/05-optimizing/index.mdx

```diff
@@ -30,7 +32,9 @@ Additionally, you can create dynamic Open Graph Images using JSX and CSS with [i
 </AppOnly>
 
 <PagesOnly>
+
 The Head Component in Next.js allows you to modify the `<head>` of a page. Learn more in the [Head Component](/docs/pages/building-your-application/optimizing/head) documentation.
+
 </PagesOnly>
```

Comment:
I was checking all the files for broken links because of the link-check failing. I didn't check all the files yet, but this is one of the files that failed. It passed when I removed the spacing between the PagesOnly component.